### DISCUSSION TITLE
moving Db2 database from Windows to Linux - does the automation framework offer any scripts ?

### DISCUSSION THREAD

**Discussion by MagnaFrisia90** [HCL-TECH-SOFTWARE/connections-automation#344] [Q&A] [answered]
Hello,

in the past I build up most of my Connections environments on Windows, manually performing GUI-install.
In most cases I used Db2 on Windows.
Now that the Connections automation framework works pretty well I would like migrate from my Windows environment to the new AlmaLinux9 environment build by the automation framework.
In general Db2 does not support moving from Windows => Linux e.g. by just using "backup" & "restore".
Also it seems like the Connections database transfer tool (dbt) that HCL offered in the past does not exist anymore.

Does the automation framework contain migration scripts for this purpose ?

Thanks in advance !

Kind regards,
Jan

**Comment by sabrina-yee** [comment, marked as answer]
Hi @MagnaFrisia90, regrettably database migration is not the scope of our automation framework.
